### PR TITLE
Move to a GitHub action (instead of container-based) CI approach

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
Also, that GitHub action is now the "official" action for Erlang+rebar3 CI, as per EEF

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -13,12 +13,14 @@ jobs:
   ci:
     name: Run checks and tests over ${{matrix.otp_vsn}} and ${{matrix.os}}
     runs-on: ${{matrix.os}}
-    container:
-      image: erlang:${{matrix.otp_vsn}}
     strategy:
       matrix:
-        otp_vsn: [21.3, 22.3, 23.2]
+        otp_vsn: [21, 22, 23]
```
Comment (g-andrade) [inline .github/workflows/ci.yml:18]: ^ Will these always "point to the latest minor"?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -13,12 +13,14 @@ jobs:
...
         os: [ubuntu-latest]
     steps:
       - uses: actions/checkout@v2
+      - uses: erlef/setup-beam@v1.7.0
+        with:
+          otp-version: ${{matrix.otp_vsn}}
+          rebar3-version: '3.14'
```
Comment (g-andrade) [inline .github/workflows/ci.yml:25]: ^ Cool!

**Review by lrascao — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (paulo-ferraz-oliveira) [reply .github/workflows/ci.yml:18]: Latest minor+patch+whatever, respecting semver rules.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (paulo-ferraz-oliveira) [reply .github/workflows/ci.yml:25]: Yeah, the previous action (named `setup-elixir`) didn't cover `rebar3`. I revamped it, we renamed it to `setup-beam` and not it's da bomb!

**Review by g-andrade — APPROVED** [review]

**Pull request merged**